pkg/specgen/volumes.go

```diff
@@ -152,3 +152,26 @@ func GenVolumeMounts(volumeFlag []string) (map[string]spec.Mount, map[string]*Na
 
 	return mounts, volumes, overlayVolumes, nil
 }
+
+// Splits a volume string, accounting for Win drive paths
+// when running as a WSL linux guest or Windows client
+func SplitVolumeString(vol string) []string {
+	parts := strings.Split(vol, ":")
+	if !shouldResolveWinPaths() {
+		return parts
+	}
+
+	// Skip extended marker prefix if present
+	n := 0
+	if strings.HasPrefix(vol, `\\?\`) {
```

Comment:
What is a extended marker prefix? What does the `\\?\` mean for a path?

Reply:
It indicates that the following path may be greater than the historical MAX_PATH (260 char) value on Win32. Old win APIs and applications can use it to keep validation compatibility, yet still tolerate newer longer paths. We don't need this marker since we aren't passing it to windows APIS, and we don't restrict path length. So we can safely strip it. The most likely case where this would show up would be someone cutting and pasting a path printed by another program.